src/SAWScript/MGU.hs

```diff
@@ -77,25 +77,27 @@ mgu m (TyUnifyVar i) t2 = bindVar m i t2
 mgu m t1 (TyUnifyVar i) = bindVar m i t1
 mgu m r1@(TyRecord ts1) r2@(TyRecord ts2) = do
   assert (M.keys ts1 == M.keys ts2) $
-    "mismatched record fields: " ++ pShow r1 ++ " and " ++ pShow r2
+    "Record Field Type Mismatch, expected " ++ pShow r1 ++ " but got " ++ pShow r2 ++ " at " ++ show m
```

Comment:
It's a minor thing, but I'd write this as `"Record field type mismatch. Expected "`. The title case looks sort of weird to me.

Reply:
And the same comment applies below, too.